Source/WebCore/svg/SVGAnimationElement.cpp

```diff
@@ -57,28 +57,27 @@ SVGAnimationElement::SVGAnimationElement(const QualifiedName& tagName, Document&
 {
 }
 
-static void parseKeyTimes(StringView parse, Vector<float>& result, bool verifyOrder)
+static Vector<float> parseKeyTimes(StringView value, bool verifyOrder)
 {
-    result.clear();
-    bool isFirst = true;
-    for (auto timeString : parse.split(';')) {
+    auto keyTimes = value.split(';');
+    Vector<float> result;
+
+    for (auto keyTime : keyTimes) {
+        keyTime = keyTime.stripWhiteSpace();
```

Comment:
This is anlmost always the wrong function to use. It follows the Unicode definition of white space. Instead we want the one that follow the SVG definition which includes far fewer characters. Possibly the ones that say “HTML” in their names are the correct ones to use.

Can also make a test for this by selecting some exotic Unicode whit space character and testing that it is not stripped.

Reply:
is `stripLeadingAndTrailingHTMLSpaces()` the correct one to use?

Reply:
It might be. We have to check that SVG's definition of space is the same as HTML. Also have to make sure we use the one that takes and returns a `StringView`. It's possible that the correct one is a template function including the template argument `<isSVGSpace>`.